pkg/networkservice/common/excludedprefixes/server.go

```diff
@@ -44,26 +47,35 @@ type excludedPrefixesServer struct {
 
 func (eps *excludedPrefixesServer) init() {
 	logger := trace.Log(eps.ctx)
-	updatePrefixes := func(bytes []byte) {
+	onUpdate := func(_ fsnotify.Event) error {
+		bytes, err := ioutil.ReadFile(filepath.Clean(eps.configPath))
```

Comment:
Please use `event.Name` because in future excludedPrefixesServer can listen to a few files.

Reply:
Woop! I've tried in and double checked - `event.Name` is a relative path to the file :(
So we really can't do nothing with this...

Reply:
Do not think so. 
1. You can look at os.Executable...
2. You can look at https://golang.org/src/path/filepath/path.go?s=7016:7053#L232

Reply:
FIxed